platform/macos/common.cmake

```diff
@@ -29,6 +30,57 @@ install(
     WORLD_READ             WORLD_EXECUTE 
 )
 
+install(
+  FILES
+    "${OSQUERY_DATA_PATH}/control/Info.plist"
+    "${OSQUERY_DATA_PATH}/control/PkgInfo"
+    "${OSQUERY_DATA_PATH}/control/embedded.provisionprofile"
+
+  DESTINATION
+    "/opt/osquery.app/Contents"
+
+  COMPONENT
+    osquery
+)
+
+install(
+  FILES
+    "${OSQUERY_DATA_PATH}/opt/bin/osqueryctl"
```

Comment:
`/opt/bin` ?

Reply:
This is just the `package_data` holding place